inference-engine/src/plaidml_plugin/ops/lstm_cell.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <limits>
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset4.hpp"
+#include "plaidml/op/op.h"
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+edsl::Tensor clip_Tensor(bool should_clip, float clip, const edsl::Tensor& T) {
+    if (should_clip) {
+        return op::clip(T, edsl::Tensor(-clip), edsl::Tensor(clip));
+    } else {
+        return T;
+    }
+}
+
+edsl::Tensor activation_func(std::string func_name, const edsl::Tensor& T) {
+    if (func_name == "relu") {
+        return op::relu(T);
+    } else if (func_name == "sigmoid") {
+        return op::sigmoid(T);
+    } else if (func_name == "tanh") {
+        return edsl::tanh(T);
+    } else {
+        THROW_IE_EXCEPTION << "Unsupported activation function";
+    }
+}
```

Comment:
Maybe we can combine `clip_Tensor()` and `activation_func()` into one function and move it to `plaidml_util.cpp` as
```cpp
edsl::Tensor clip_activation(std::string activation, const edsl::Tensor& T, bool should_clip, float clip) {
    edsl::Tensor T_clipped;
    if (should_clip) {
        T_clipped = op::clip(T, edsl::Tensor(-clip), edsl::Tensor(clip));
    } else {
        T_clipped = T;
    }
    if (func_name == "relu") {
        return op::relu(T_clipped);
    } else if (func_name == "sigmoid") {
        return op::sigmoid(T_clipped);
    } else if (func_name == "tanh") {
        return edsl::tanh(T_clipped);
    } else {
        THROW_IE_EXCEPTION << "Unsupported activation function";
    }
}
```

Reply:
I think for now, just put these in an unnamed namespace in this file.

But, if we need these anywhere else, then yes, we should do as you suggest, and move these into `plaidml_util.cpp` (or perhaps even the PlaidML oplib)

Reply:
From my current observation, these works for `RNNCell`, `GRUCell` and `LSTMCell`. Not sure in other ops, if there are any other activation functions will be included.
Maybe we can keep these in op file and decide whether put them into `plaidml_util.cpp` after going through most ops.

Reply:
Ok, since it's used for those 3, let's move it to plaidml_util.cpp

Reply:
I have move it to plaidml_util.cpp. Once this PR has been merged, I'll apply this change to `RNNCell` and `GRUCell`.